saleor/webhook/payloads.py

```diff
@@ -581,6 +581,39 @@ def generate_list_gateways_payload(
     return json.dumps(payload)
 
 
+def generate_shipping_methods_payload(checkout: Optional["Checkout"]):
```

Comment:
Can't we use logic which generates payload for order and checkout? I think that it is better to extend them than adding new function for generating payload. Espeicially that we will need two functions. for order and for checkout